scripts/http-form-fuzzer.nse

```diff
@@ -7,6 +7,7 @@ determine if the fuzzing was successful.
 ---
 -- @usage
 -- nmap --script http-form-fuzzer --script-args 'http-form-fuzzer.targets={1={path=/},2={path=/register.html}}' -p 80 <host>
+-- nmap --script http-form-fuzzer --script-args 'http-form-fuzzer.timelimit=50m','http-form-fuzzer.minlength=100000' -p 80 <host>
```

Comment:
Let's not break the quoting in the middle of script-args in the example, like `','`. Instead, just keep the quotes at the beginning and end.